commit 70561be11a040ce7ce2fd7274306357ceae91e97
Author: haolong.zhang <haolong.zhang@ck-telecom.com>
Date:   Thu Oct 9 18:27:20 2014 +0800

    [BugFix][TP] tmp close DMA MODE
    
    [id]::66937
    [Type]
       BUG
    
    [Products]
    	EYES_BASELINE
    [Requirement]
       N/A
    
    [Solution Description]
       N/A
    
    [Other Info]
       N/A
    
     要提交的变更：
    	修改:         mediatek/custom/ckt95_lw_kk/kernel/touchpanel/ektf2k/tpd_custom_ektf2k.h
    
    Change-Id: Iadea38330597bbf05180c431856faa35f354ca85

diff --git a/mediatek/custom/ckt95_lw_kk/kernel/touchpanel/ektf2k/tpd_custom_ektf2k.h b/mediatek/custom/ckt95_lw_kk/kernel/touchpanel/ektf2k/tpd_custom_ektf2k.h
index 03363b9..00b4cb3 100755
--- a/mediatek/custom/ckt95_lw_kk/kernel/touchpanel/ektf2k/tpd_custom_ektf2k.h
+++ b/mediatek/custom/ckt95_lw_kk/kernel/touchpanel/ektf2k/tpd_custom_ektf2k.h
@@ -19,7 +19,7 @@
 
 //#define SOFTKEY_AXIS_VER
 //#define ELAN_TEN_FINGERS
-#define _DMA_MODE_
+//#define _DMA_MODE_
 
 #define ELAN_BUTTON
 #define LCT_VIRTUAL_KEY
